<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OOMetrics</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QTextEdit" name="plainTextEdit">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="Metrics">
            <property name="text">
             <string>Посчитать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openfile">
            <property name="text">
             <string>Открыть файл</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="Lable_KolMet">
                  <property name="text">
                   <string>Количество методов:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="Kol_Met">
                  <property name="text">
                   <string>_</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="Label_KolMod">
                  <property name="text">
                   <string>Количество модулей:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="Kol_Mod">
                  <property name="text">
                   <string>_</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="Include">
                  <property name="text">
                   <string>_</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="Label_KolClass">
                  <property name="text">
                   <string>Количество классов:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="Kol_Class">
                  <property name="text">
                   <string>_</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Среднее количество строк методов:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_SrKSMet">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Среднее количество строк модулей:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_SrKSMod">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Среднее количество строк классов:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_SrKSCl">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Суммарная сложность методов класса (WMC):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_WMC">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Глубина дерева наследования (DIT):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_DIT">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Количество непосредственных наследников (NOC):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_NOC">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Количество методов в множестве отклика (RFC):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_RFC">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Недостаток сцепления методов (LCOM):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_LCOM">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Количество вызовов оттдаленных методов (NORM):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_NORM">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
